demo/web/src/main/notebooks/B. Data science, ML, & AI/B3 TensorFlow and Deephaven.md

```diff
@@ -10,11 +10,12 @@ Let's start by importing everything we need.  We split up the imports into three
 
 ```python
 # Deephaven imports
-from deephaven import dataFrameToTable, tableToDataFrame
-from deephaven import DynamicTableWriter
-from deephaven import learn, read_csv
-from deephaven.learn import gather
-import deephaven.Types as dht
+from deephaven2.pandas import to_pandas, to_table
+from deephaven2 import DynamicTableWriter
+from deephaven2 import dtypes as dht
```

Comment:
Just a note that this differs from `import deephaven2.dtypes as dht` that you have above. I don't think it matters though, only pointing out in case you want consistency